app/views/katello/api/v2/content_facet/show.json.rabl

```diff
@@ -0,0 +1,11 @@
+child :content_facet => :content_attributes do
```

Comment:
I am good with those. I tend to agree 'attributes' is superfluous, and including facet brings an implementation detail to the user that they would otherwise not know or care about.